toolshed/env.ts

```diff
@@ -4,6 +4,7 @@ import * as Path from "@std/path";
 // NOTE: This is where we define the environment variable types and defaults.
 const EnvSchema = z.object({
   ENV: z.string().default("development"),
+  HOST: z.string().default("0.0.0.0"),
```

Comment:
nit: not sure where/when this applies, but I think we'd want `127.0.0.1` for local dev (local connections only), `0.0.0.0` for a server host

Reply:
done!